torch/_meta_registrations.py

```diff
@@ -1636,3 +1638,64 @@ def upsample_nearest2d_vec(input, output_size, scale_factors):
+        assert isinstance(op_overload, OpOverload)
+
+        op_overload.py_impl(torch._C.DispatchKey.Meta)(fn)
+
+        # TODO: factor this logic into OpOverload or Library API
+        name = op_overload._schema.name
+        if op_overload._schema.overload_name:
+            name += "." + op_overload._schema.overload_name
+
+        dispatch_key_registration = torch._C._dispatch_dump(name)
+
+        if "CompositeImplicitAutograd" in dispatch_key_registration:
+            # Internally, we shouldn't be registering meta kernels for any operators that
+            # have CompositeImplicitAutograd kernels.
+            # Instead, we should be letting those decompositions run, and writing meta kernels
+            # only for the base operators.
+            pass
+        elif any(
+            a.alias_info is not None and not a.alias_info.is_write
+            for a in op_overload._schema.arguments
+        ):
+            # Attempting to register a python meta kernel for a view operator.
+            # We shouldn't do this, because the output will report as not having aliased storages.
+            # All view ops have meta kernels in C++ today, so we should use those instead.
+            pass
+        elif name in {
+            "aten::empty_strided",  # causing infinite recursion, test/test_meta.py
+            "aten::clone",  # causing infinite recursion,
+            "aten::_to_copy",  # causing infinite recursion, test/test_serialization.py -k test_tensor_subclass_getstate_overwrite
+            "aten::randn",  # pin_memory parameter is not supported!, test/test_proxy_tensor.py -k test_make_fx_symbolic_exhaustive_randn_cpu_float32
+            "aten::zeros.names",  # TypeError: zeros() got an unexpected keyword argument 'names', test/inductor/test_torchinductor.py -k test_zeros_cpu
+            "aten::empty.names",  # TypeError: empty() got an unexpected keyword argument 'names', test/inductor/test_torchinductor.py -k test_zeros_cpu
+            "aten::add.Tensor",  # ValueError: Receive two Number inputs to an elementwise binary operation! test/inductor/test_torchinductor.py -k test_both_scalars
+            "aten::sub.Tensor",  # ValueError: Receive two Number inputs to an elementwise binary operation! test/inductor/test_torchinductor.py -k test_both_scalars
+            "aten::mul.Tensor",  # ValueError: Receive two Number inputs to an elementwise binary operation! test/inductor/test_torchinductor.py -k test_both_scalars
+            "aten::div.Tensor",  # ValueError: Receive two Number inputs to an elementwise binary operation! test/test_fake_tensor.py -k test_scalar_inputs
+            "aten::div.Tensor_mode",  # ValueError: Receive two Number inputs to an elementwise binary operation! test/inductor/test_torchinductor.py -k test_div8_cpu
+            "aten::diag_embed",  # RuntimeError: Stride mismatch! Strides are (180, 30, 1, 6) and (180, 30, 5, 1) (mismatched at 2)! test/test_ops.py -k test_fake_autocast_diag_embed_cuda_float32
```

Comment:
it seems like we're just moving the public api into a private list. Is that useful ?

Reply:
This is a list I will work to burn down. 
Most of them can be resolved by fixing the meta implementation. 

For the ones that python meta doesn't work, I plan to add it to the block list in functions_with_cpp_meta_impl_that_support_symint. 

So eventually, it should just look up from functions_with_cpp_meta_impl_that_support_symint.